package.json

```diff
@@ -65,6 +64,7 @@
     "@opentelemetry/semantic-conventions": "1.22.0",
     "ajv": "8.13.0",
     "commander": "12.0.0",
+    "drizzle-orm": "^0.33.0",
```

Comment:
We shouldn't need this as a direct dependency, just a dev dependency. Can you revert it to a dev dependency and verify tests still pass?